README.md

```diff
@@ -19,32 +19,16 @@ Reference headless ecommerce storefront for [Reaction Commerce](https://reaction
 
 ## Getting Started
 
-1. Requirements:
-
-- Install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/). Docker Compose is included when installing Docker on Mac and Windows, but will need to be installed separately on Linux.
-- Install [Node.js](https://nodejs.org/en/)
-- Install [Reaction Platform](https://github.com/reactioncommerce/reaction-platform)
-- Make sure you are not running any applications on ports `3000` and `4000`.
-
-2. Follow the instructions at [Reaction Platform](https://github.com/reactioncommerce/reaction-platform) to run Reaction, Reaction Hydra and the starterkit with Reaction Platform.
-
-After the initial bootstrapping, you can use `make start` to start all the necessary containers:
-
-```sh
-make start
-```
-
-3. Once the `make` process finishes, the Reaction Platform services will be accessible at the following URLs:
-
-| Application  | URL                     |
-| ------------ | ----------------------- |
-| `starterkit` | localhost:4000          |
-| GraphQL API  | localhost:3000/graphiql |
-| `reaction`   | localhost:3000          |
-
-Visit the storefront at `localhost:4000` and the GraphQL API playground at `localhost:3000/graphiql`.
-
-4. To view the logs for the Starterkit, run: `docker-compose logs -f`
+Follow the [Reaction Platform docs](https://docs.reactioncommerce.com/docs/installation-reaction-platform) to install and run all the services started to run the Starter Kit:
```

Comment:
"all the services started to run" sounds strange. Is "started" supposed to be "necessary"?

Reply:
Looks good